demo/src/main/java/com/novoda/demo/MainActivity.java

```diff
@@ -17,14 +19,16 @@
 
 public class MainActivity extends Activity {
 
-    private static final String URI_VIDEO_WIDEVINE_EXAMPLE_MODULAR_MPD = "https://storage.googleapis.com/content-samples/multi-audio/manifest.mpd";
+    private static final String URI_VIDEO_WIDEVINE_EXAMPLE_MODULAR_MPD = "https://storage.googleapis.com/wvmedia/clear/h264/tears/tears.mpd";
```

Comment:
I had to change this since the MPD we were using only published one video `Representation`.